dndwiki/races/race-concubus.xml

```diff
@@ -0,0 +1,98 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<elements>
+	<info>
+		<name>Concubus</name>
+		<description></description>
+		<author url="https://www.dandwiki.com/wiki/User:Geodude671">Geodude671</author>
+		<update version="0.0.1">
+			<file name="race-concubus.xml" url="https://raw.githubusercontent.com/community-elements/elements-dndwiki/master/dndwiki/races/race-concubus.xml" />
+		</update>
+	</info>
+
+  <element name="Concubus" type="Race" source="D&amp;D Wiki" id="ID_WIKI_RACE_CONCUBUS">
```

Comment:
Why is the race named Concubus instead of Succubus? Unless it's an obvious typo stick as close to the source as possible.